### DISCUSSION TITLE
Subset seurat object

### DISCUSSION THREAD

**Discussion by yamihn** [stuart-lab/signac#1258] [Q&A] [answered]
Hi, I want to subset my multiomic object containing RNaseq and ATAC Seq. I subset my object keeping only some clusters deriver by the integration. Next step was find variable genes, scalimg data, run PCA, reclustering. This was on RNA assay. My doubt Is, I have to run LSI again for the ATAC part, and subset the fragments? For the Rna assay Is quite clear and documented, but I don't how to move forward, because Im interested to find DARs in the new subsetted and reclustered seurat obj

**Comment by timoast** [comment, marked as answer]
For differential accessibility testing there's no need to re-run LSI after subsetting

**Reply by yamihn** [reply]
<thank you @timoast . I've found a higher number of dars comparing KO vs WT in specific timepoints in my subsetted and reclustered object. I don't know if I should expect this. In the whole object, the number of dars for each cluster comparing KO vs WT at each timepoint was about 20. Now I can found at least 500 DARs in some clusters

**Reply by timoast** [reply]
I can't really give advice as to what's expected for your experiment
